opal/mca/btl/tcp/btl_tcp_endpoint.c

```diff
@@ -62,6 +62,10 @@
 #include "btl_tcp_frag.h"
 #include "btl_tcp_addr.h"
 
+/*
+ * Magic ID string send during connect/accept handshake
+ */
+const char mca_btl_tcp_magic_id_string[] = "OPAL-TCP-BTL";
```

Comment:
We are targeting on verifying established connection is from MPI process. We are not verifying its belonging to same version or job here. Can add one new PR for it if it's one of the failure case we face today